src/transformers/models/oneformer/image_processing_oneformer.py

```diff
@@ -386,11 +402,11 @@ class OneFormerImageProcessor(BaseImageProcessor):
             Whether or not to decrement all label values of segmentation maps by 1. Usually used for datasets where 0
             is used for background, and background itself is not included in all classes of a dataset (e.g. ADE20k).
             The background label will be replaced by `ignore_index`.
-        repo_path (`str`, defaults to `shi-labs/oneformer_demo`, *optional*, defaults to `"shi-labs/oneformer_demo"`):
-            Dataset repository on huggingface hub containing the JSON file with class information for the dataset.
+        repo_path (`str`, *optional*, defaults to `"shi-labs/oneformer_demo"`):
+            Path to hub repo or local directory containing the JSON file with class information for the dataset.
+            If unset, will load `class_info_file` from the current working directory.
```

Comment:
Could be more precise: if unset or the corresponding file could not be found locally ...

Reply:
If unset, then `repo_path` is set to `""` - effectively making the filename `class_info_file` rather than `repo_path/class_info`, but it's independent of whether it can be found locally. 

Would this be clearer: 
```
            Path to hub repo or local directory containing the JSON file with class information for the dataset.
            If unset, will look for `class_info_file` in the current working directory.
```

And then adding an additional check in `load_metadata`:

```

def load_metadata(repo_id, class_info_file):
    fname = os.path.join("" if repo_id is None else repo_id, class_info_file)

    if not os.path.exists(fname) or not os.path.isfile(fname):
        if repo_id is None:
            raise ValueError(f"Could not file {fname} locally. repo_id must be defined if loading from the hub")
```

Reply:
Either works for me, but we can go for the above one  for simplicity.

Reply:
I did both in [a777668](https://github.com/huggingface/transformers/pull/28398/commits/a77766833b865cd4b0b5d3c3c9ac74174c168bd6) :)